cocos/scripting/lua-bindings/manual/LuaBasicConversions.cpp

```diff
@@ -846,7 +846,7 @@ bool luaval_to_fontdefinition(lua_State* L, int lo, FontDefinition* outValue , c
         lua_gettable(L,lo);
         if (!lua_isnil(L,-1))
         {
-            luaval_to_color3b(L, -1, &outValue->_fontFillColor);
+            luaval_to_color3b(L, lua_gettop(L), &outValue->_fontFillColor);
```

Comment:
I'm sorry.This is really a bug.The -1 index would be represent the different top index.For those table parameter conversion,we should use the top index, Thanks to @htlxyz.